script/Arcueid/Day 11/11_00_ARC11_2.txt

```diff
@@ -751,42 +812,46 @@ The 12th century, when she was born, was an era of fallen True Ancestors running
 [sha:b565f68f6e7a274050a79c770a574f9919aa2836]{
 -- Page 92, Offset 29758.
 -- 「真祖たちは堕ちた真祖も、その真祖が<徒|いたずら>に増やしていく死徒も放ってはおけなかった。
-"The True Ancestors couldn't leave their fallen counterparts and the army of frivolously created Dead Apostles they'd sired alone.
+"The True Ancestors couldn't leave their fallen counterparts―――and the armies of Dead Apostles they'd sired―――alone.
 }
 [sha:100f40e1f487f98ff780b4b9c3598f2d7fddea29]{
 -- Page 92, Offset 29759.
 -- その手段として、ただ殺す為だけの、それ以外の事を何も必要としない純粋な真祖を誕生させ、処刑役にした」
-So for that――for the sole purpose of killing, without any other duty――a pure True Ancestor was born as an executioner."
+To that end they gave birth to a pure True Ancestor,
+ one who existed for the sole purpose of killing,
+ who could serve as their executioner."
```

Comment:
maybe "an executioner". "their executioner" sounds like her role is to execute the TAs who sired her (though i guess she _did_ do that).